pkg/minikube/assets/vm_assets.go

```diff
@@ -104,6 +111,15 @@ func (f *FileAsset) GetLength() (flen int) {
 	return int(fi.Size())
 }
 
+// GetModTime returns modification time of the file
+func (f *FileAsset) GetModTime() (time.Time, error) {
+	fi, err := os.Stat(f.AssetName)
+	if err != nil {
```

Comment:
You can likely get away with returning `fi.ModTime(), err` directly instead of checking for err.

Reply:
done